bpfd/src/server/bpf.rs

```diff
@@ -267,7 +267,12 @@ impl<'a> BpfManager<'a> {
                     let iov = [IoSlice::new(b"a")];
                     let fds = [fd.as_raw_fd()];
                     let cmsg = ControlMessage::ScmRights(&fds);
-                    sendmsg(sock, &iov, &[cmsg], MsgFlags::empty(), Some(&sock_addr)).unwrap();
+                    if let Err(result) =
+                        sendmsg(sock, &iov, &[cmsg], MsgFlags::empty(), Some(&sock_addr))
```

Comment:
Something I found out the other day is that you could use `sendmsg(sock, &iov, &[cmsg], MsgFlags::empty(), Some(&sock_addr)).map_error(BpfdError::NotAuthorized)?` 

If you wanted to one line it, but you'd lose that `info!` message

Reply:
[`map_err`](https://doc.rust-lang.org/std/result/enum.Result.html#method.map_err) is totally fine,
the info log message shouldn't be needed...

we probably need to log these errors when they occur from elsewhere in bpfd (i.e in `rpc.rs` before we return them to the client)

Reply:
I had a scenario where `gocounter` created a socket and had access to it and `bpfd` did not (I was still working through all the permissions). Without the info log, it just fails silently. I don't know how many other scenarios that will occur, but I think it is helpful. I do need to change the returned error. I just picked one, but I'll create a new BpfdError::SendError.